HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -48,24 +72,43 @@ public final class PaymentComponentsController: NSObject {
         }
     }
 
-    var isLoading: Bool = false {
+    /// reponsible for storing the loading state of the controller and passing it to the delegate listeners
+    private var isLoading: Bool = false {
         didSet {
             delegate?.isLoadingStateChanged(isLoading: isLoading)
         }
     }
 
-    var paymentComponentView: PaymentComponentView!
-
+    /**
+     Returing the first payment provider installed if is available and installed
```

Comment:
[suggestion]: `Retrieve the first installed payment provider, if available.` instead of ` Returing the first payment provider installed if is available and installed` (typo `Returing` -> `Returning`)